trading_calendars/exchange_calendar_six.py

```diff
@@ -0,0 +1,149 @@
+#
+# Copyright 2018 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import time
+from pandas.tseries.holiday import (
+    Easter,
+    EasterMonday,
+    GoodFriday,
+    Holiday,
+)
+from pandas.tseries.offsets import Day
+from pytz import timezone
+
+from .trading_calendar import (
+    TradingCalendar,
+    HolidayCalendar,
+)
+
+# Regular Holidays
+# ----------------
+SIXNewYearsDay = Holiday(
+    "New Year's Day",
+    month=1,
+    day=1,
+)
+
+BerchtoldsDay = Holiday(
+    "Berchtold's Day",
+    month=1,
+    day=2,
+)
+
+LabourDay = Holiday(
+    "Labour Day",
+    month=5,
+    day=1,
+)
+
+AscensionDay = Holiday(
+    "Ascension Day",
+    month=1,
+    day=1,
+    offset=[Easter(), Day(39)],
+)
+
+WhitMonday = Holiday(
+    "Whit Monday",
+    month=1,
+    day=1,
+    offset=[Easter(), Day(50)],
+)
+
+SwissNationalDay = Holiday(
+    "Swiss National Day",
+    month=8,
+    day=1
+)
+
+ChristmasEve = Holiday(
```

Comment:
These seem like good candidates for a common holidays file as well.

Reply:
Seems like everything here but "Berchtold's Day" and "Swiss National Day" could probably be shared.

Reply:
(I'd probably rename labour day to something like "European Labour Day" though if we move it to a common location.)

Reply:
👍 moved a bunch of stuff to common_holidays, and removed their definitions in the specific calendars